napari/layers/image/image.py

```diff
@@ -689,3 +689,8 @@ def contrast_limits(self, contrast_limits):
                 self.refresh()
             finally:
                 self._keep_auto_contrast = prev
+
+    def _get_first_valid_indices(self, values):
+        return np.flatnonzero(
+            np.where(values > self.contrast_limits[0], values, 0)
+        )
```

Comment:
I don't think this is right for images? — I think the image value returned should depend on the rendering mode? (ie for MIP it should return the maximum value along the ray, etc.)

Reply:
Good point!

Reply:
It did seem to easy :P

Reply:
fixed!